server/helpers/mailer.js

```diff
@@ -0,0 +1,85 @@
+import sgMail from '@sendgrid/mail';
+import Mailgen from 'mailgen';
+import { config } from 'dotenv';
+
+config();
+
+const url = process.env.BASE_URL || 'https://kingsmen-ah-frontend-staging.herokuapp.com';
+const projectName = process.env.PROJECT_NAME || 'Author\'s Haven';
+const projectEmail = process.env.PROJECT_EMAIL || 'authorshaven@gmail.com';
+
+sgMail.setApiKey(process.env.SENDGRID_API_KEY);
+
+// Configure the mail gen
+const mailGenerator = new Mailgen({
+  theme: 'cerberus',
+  product: {
+    name: projectName,
+    link: url
+  }
+});
+
+const sendMail = ({ to, subject, message }) => {
+  const mailOptions = {
+    from: `${projectName} <${projectEmail}>`,
+    to,
+    subject,
+    html: message
+  };
+
+  sgMail.send(mailOptions);
+};
+
+const sendForgotPasswordMail = (token, email, name) => {
+  const link = `${url}/auth/reset_password?email=${email}&token=${token}`;
+  const emailBody = {
+    body: {
+      name,
+      intro: 'You are receiving this email because a password reset request for your account was received.',
+      action: {
+        instructions: 'Tap the button below to reset your customer account password. If you didn\'t request a new password, you can safely delete this email.',
```

Comment:
Does eslint throw an error when you do this?
```suggestion
        instructions: `Tap the button below to reset your customer account password. If you didn't request a new password, you can safely delete this email.`,
```

Reply:
Yes it does "Strings must use singlequote."

Reply:
Alright, makes sense.